.eslintignore

```diff
@@ -1,6 +1,7 @@
-node_modules
-dist
+node_modules/
+dist/
```

Comment:
Was this necessary?

Reply:
If not necessary, that's fine; I'm just curious if some environment behaves differently or what not and necessitated this.

Reply:
Not sure. probably some extension did this.
I'll revert.